File: src/main/java/org/springframework/samples/petclinic/HelloWorld.java
package org.springframework.samples.petclinic;

import org.eclipse.jetty.server.Server;
import org.eclipse.jetty.server.ServerConnector;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

class HelloWorld {
    private static final Logger logger = LoggerFactory.getLogger(HelloWorld.class);

    public static void main(String[] args) throws Exception {
        logger.debug("It is starting the application");
        final Server server = new Server();
        ServerConnector connector = new ServerConnector(server);
        connector.setPort(8079);
        server.setConnectors(new ServerConnector[]{connector});
        server.setHandler(new JettyHandler());
        server.start();
        logger.debug("The application is ready");
    }
}

File: src/main/kotlin/org/springframework/samples/petclinic/HelloWorld.kt
package org.springframework.samples.petclinic

import mu.KotlinLogging
import org.eclipse.jetty.server.Server
import org.eclipse.jetty.server.ServerConnector
import org.eclipse.jetty.server.handler.HandlerCollection

private val logger = KotlinLogging.logger {}
@Throws(Exception::class)
fun main(args: Array<String>) {
    logger.debug { "It is starting the application" }
    val server = Server()
    val connector = ServerConnector(server)
    connector.port = 8078
    server.connectors = listOf(connector).toTypedArray()
    server.handler = JettyHandler()
    server.start()
    logger.debug { "The application is ready" }
}
